_data/projects/PetMe.yml

```
@@ -0,0 +1,11 @@
+name: PetMe
+desc: PetMe is an all-in-one platform that allows animals to be adopted, donated to pet lovers, and provides emergency medical care to stray animals in need.
+site: https://akshitagupta15june.github.io/PetMe/
+tags:
+  - html
+  - css
+  - javascript
+  - postman-api
+upforgrabs:
+  name: good first issue
+  link: https://github.com/akshitagupta15june/PetMe/issues?q=is%3Aissue+is%3Aopen+label%3A%22good+first+issue%22
```

:art: let's tidy this up to a shorter URL:

```suggestion
  link: https://github.com/akshitagupta15june/PetMe/labels/good%20first%20issue
```